src/partest/scala/tools/partest/nest/Runner.scala

```diff
@@ -462,14 +459,11 @@ class Runner(val testInfo: TestInfo, val suiteRunner: AbstractRunner) {
     state
   }
 
-  // snort or scarf all the contributing flags files
+  // all sources in a round may contribute flags via // scalac: -flags
```

Comment:
You've lost some good words here.

Reply:
`slurp`; but there are more where those came from.